modular_skyrat/modules/ghostcafe/code/robot_ghostcafe.dm

```diff
@@ -39,15 +39,29 @@
 		/obj/item/stack/cable_coil,
 		/obj/item/restraints/handcuffs/cable/zipties,
 		/obj/item/rsf/cyborg,
+		/obj/item/reagent_containers/borghypo/borgshaker/specific/juice, // They can make glasses, and have their own, let them serve drinks!
+		/obj/item/reagent_containers/borghypo/borgshaker/specific/soda,
+		/obj/item/reagent_containers/borghypo/borgshaker/specific/alcohol,
+		/obj/item/reagent_containers/borghypo/borgshaker/specific/misc,
 		/obj/item/reagent_containers/food/drinks/drinkingglass,
 		/obj/item/soap/nanotrasen,
+		/obj/item/mop/cyborg, // Soap's good and all, but a mop is good, too
+		/obj/item/lightreplacer, // Lights go out sometimes, or get broken, let the Borg help fix them
 		/obj/item/borg/cyborghug,
 		/obj/item/dogborg_nose,
 		/obj/item/dogborg_tongue,
+		/obj/item/reagent_containers/borghypo, // Let Roleplay Borgs heal visitors
```

Comment:
if visitors get injured they can just ghost and respawn tho

Reply:
Yeah but they can also get healed by the borg.

I have no problem with this.